src/avatar/Avatar.tsx

```diff
@@ -0,0 +1,35 @@
+import "./_avatar.scss";
+
+import React, {useState} from "react";
+import classNames from "classnames";
+
+export interface AvatarProps {
+  alt: string;
+  size?: {
+    width: string;
+    height: string;
+  };
+  placeholderSrc: string;
+  src?: string | null;
+  customClassName?: string;
```

Comment:
```suggestion
  placeholderSrc: string;
  size?: {
    width: string;
    height: string;
  };
  src?: string | null;
  customClassName?: string;
```
Let's group optional props at the bottom to follow our conventions